changedetectionio/content_fetchers/base.py

```diff
@@ -113,14 +113,14 @@ def get_all_headers(self):
     def browser_steps_get_valid_steps(self):
         if self.browser_steps is not None and len(self.browser_steps):
             valid_steps = filter(
-                lambda s: (s['operation'] and len(s['operation']) and s['operation'] != 'Choose one' and s['operation'] != 'Goto site'),
-                self.browser_steps)
+                lambda s: (s['operation'] and len(s['operation']) and s['operation'] != 'Choose one'),
+                self.browser_steps[1:]) # Skip the first one, it is already handled before the browser steps are run
 
             return valid_steps
 
         return None
 
-    def iterate_browser_steps(self):
+    def iterate_browser_steps(self, url):
```

Comment:
seems confusing and vague why this has `url` here

can you rename it to something like `start_url` ? or? im confused

Reply:
Done.